hack/cluster-deploy.sh

```diff
@@ -58,9 +58,7 @@ function _ensure_cdi_deployment() {
     _kubectl patch cdi ${cdi_namespace} --type merge -p '{"spec": {"config": {"uploadProxyURLOverride": "'"$override"'"}}}'
 
     # Enable succeeded DataVolume garbage collection
-    if [[ $CDI_DV_GC != "false" ]]; then
-        _kubectl patch cdi ${cdi_namespace} --type merge -p '{"spec": {"config": {"dataVolumeTTLSeconds": '"$CDI_DV_GC"'}}}'
-    fi
```

Comment:
Let's keep this if statement for people (like yourstruly) that want to disable GC

Reply:
So set CDI_DV_GC=-1 when you want to disable it